controller/workitem_whitebox_test.go

```diff
@@ -164,6 +164,59 @@ func (rest *TestWorkItemREST) TestConvertJSONAPIToWorkItemWithLegacyDescription(
 
 }
 
+// TODO(Ibrahim) - This test should be removed after we remove support for system.* fields
+func (rest *TestWorkItemREST) TestConvertJSONAPIToWorkItemWithDotAndPeriodFields() {
+	resource.Require(rest.T(), resource.Database)
+	//given
+	fxt := tf.NewTestFixture(rest.T(), rest.DB, tf.CreateWorkItemEnvironment())
+	rest.T().Run("system.* to system_*", func(t *testing.T) {
+		attributes := map[string]interface{}{
+			"system.title":   "title",
```

Comment:
I understand why you don't use constants here but literal stings. But it reminds me: Have you searched for other places in the code where we do this, I mean manually assign field names using string literals?